apps/src/sites/studio/pages/utils.js

```diff
@@ -0,0 +1,12 @@
+import $ from 'jquery';
+
+/*
+ * Add tooltip toggles to vocabulary definitions, as generated by the
+ * MarkdownPreprocessor
+ * @see https://getbootstrap.com/2.3.2/javascript.html#tooltips
+ */
+export function tooltipifyVocabulary() {
+  $('.vocab').each(function() {
+    $(this).tooltip({placement: 'bottom'});
+  });
+}
```

Comment:
how about apps/src/utils/ for the location of this file? AFAIK we only put entry points in apps/src/sites , and this file would be the first to break that invariant.